### DISCUSSION TITLE
Windows 10, how to minimize localsend in tray?

### DISCUSSION THREAD

**Discussion by paolosezart** [localsend/localsend#1174] [Discussion (English, International)]
How to minimize localsend to tray in Windows10? If I click the "minimize" button, the program minimizes to the taskbar and takes up space there. If I click the "close" button, the localsend program closes completely. Is it possible to configure the program so that it can be minimized to the notification area in Windows10?

And is it possible to run the program as a Windows service, i.e. without the user logging in?

**Comment by nvnth** [comment]
there's a toggle in settings to minimise
![Screenshot (2)](https://github.com/localsend/localsend/assets/87908095/f3440ed9-5ff5-4e90-b0ba-6ee3438d5909)

**Reply by ed65love** [reply]
Yes there is, but it doesn't work under the latest version of w11.

**Reply by harrischrisa** [reply]
If you use the X button the windows is minimised to the task bar/notification area.  If you use the - icon the window is minimised to the system bar

**Reply by ed65love** [reply]
Yes, that's how it's supposed to work. All programs under windows. 

But it doesn't work here with this. I click X, it's gone, nothing to be seen anywhere.

i.e. that switch doesn't work :(

I have to rerun it again if I want to use it again.

That's not crucial, but it's annoying, and it's a bug.

I keep my laptop uptodate, so I don't think it's a w11 issue.

**Reply by harrischrisa** [reply]
Certainly works like this for me on Windows 11

**Reply by ed65love** [reply]
Far out. I have no idea how to debug this. i.e. what could possibly be different on our systems?

Mine isn't customised at all. I try to keep things simple.
